.github/workflows/benchmarks.yml

```diff
@@ -106,4 +106,4 @@ jobs:
     needs: prepare
     with:
       benchmark: filter-list/filter-list
-      timeout: 10
+      timeout: 10
```

Comment:
Sorry for the noise, but does seem to be a legitimate formatting correction: https://github.com/preactjs/preact/blob/3fe5d6b00fa6a7d0a1af83f2ea8822a7f82cdd6c/.editorconfig#L13